README.md

```diff
@@ -98,6 +98,70 @@ GOTIFY_PASSSTRENGTH=10
 GOTIFY_UPLOADEDIMAGESDIR=images
 ```
 
+### Examples
+---
+After creating your application, copy the `token` ( from `apps` tab )
```

Comment:
I guess it would be good to make it more specific what kind of token it is (there is a client token and an application token) the REST-API also provides functionality to obtain the app-token.
```suggestion
You can obtain an application-token from the apps tab inside the UI or using the REST-API (`GET /application`)
```